### ISSUE TITLE
Design the resource page

### ISSUE THREAD

**Issue by codecShivam** [FOSS-Community/fosscu.org#259]
For better approach, you can use dynamic routing in Next.js when designing resource-related pages.
You can refer to the previous resource pages for guidance.

**Comment by piyusesh** [comment]
@codecShivam I would like to work on this , pls assign me this

**Comment by codecShivam** [comment]
> @codecShivam I would like to work on this , pls assign me this

yeah go ahead! @piyusesh  
you can find the resources here : https://github.com/FOSS-Community/Learning-Resources

**Comment by harsh0728** [comment]
@codecShivam i can solve this and design the resource page Assign this issue to me

**Comment by codecShivam** [comment]
> @codecShivam i can solve this and design the resource page Assign this issue to me

Assigned! Feel free to reach out if needed

**Comment by harsh0728** [comment]
@codecShivam what i have to make changes in the resource page 
frontend page is complete but backend , web 3.0 , app development , ai/ml pages are incomplete 
so should i have to complete these pages??

**Comment by harsh0728** [comment]
@codecShivam  keep me update so that i can start working on this as soon as possible

**Comment by codecShivam** [comment]
Yes, you need to implement those pages, and for resources, you can visit https://github.com/fOSS-Community/learning-resources

**Comment by YASH-YADAV-dynamo** [comment]
We will be closing this issue as the site has been migrated to a new frontend, making the Web 2.0 version no longer relevant to the main platform. Thank you for your understanding.

**Comment by YASH-YADAV-dynamo** [comment]
We will be closing this issue as the site has been migrated to a new frontend, making the Web 2.0 version no longer relevant to the main platform. Thank you for your understanding.
